sql/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -213,6 +214,13 @@ public void clusterChanged(ClusterChangedEvent event) {
                 .map(UserDefinedFunctionMetaData::schema)
                 .forEach(schemas::add);
         }
+        ViewsMetaData viewsMetaData = metaData.custom(ViewsMetaData.TYPE);
+        if (viewsMetaData != null) {
+            StreamSupport.stream(viewsMetaData.names().spliterator(), false)
+                .map(RelationName::fromIndexName)
```

Comment:
Could go through `IndexParts -> getSchema` to avoid the intermediate `RelationName` instance (internally `fromIndexName` creates a `IndexParts` instance)